rover/src/science_module/scripts/science_module_listener.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+"""
+Processes data from science module topics
+@author: (Original Author) Sajid Ibne Anower
+@authors: (Editors)
+Purpose: Subscriber for science module
+@copyright: This code is released under the MIT [GPL for embeded] License.
```

Comment:
s/[GPL for embeded]// this isn't embeded :P

Reply:
Woops. Fixed